services/app-api/forms/mcpar.json

```diff
@@ -1584,6 +1584,45 @@
                             ]
                           }
                         },
+                        {
+                          "id": "program_didStateOrMCOsCompleteAnalysis",
+                          "type": "radio",
+                          "validation": {
+                            "type": "radio",
+                            "nested": true,
+                            "parentFieldName": "program_doesThisProgramIncludeMCOs",
+                            "parentOptionId": "A4uFU0YyU9useo3DsWjnH264"
+                          },
+                          "props": {
+                            "label": "C1.XII.6 Did the State or MCOs complete the analysis(es)?",
```

Comment:
Nit: Since the plural of analysis is analyses and not analysises, it's incorrect to use parentheses in this way.

Reply:
I agree, but unfortunately, that was the copy given to us by the business owners 😞